vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/model/PathfinderType.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See {@literal <https://vaadin.com/commercial-license-and-service-terms>} for the full
+ * license.
+ */
+package com.vaadin.flow.component.charts.model;
+
+/**
+ * Pathfinder algorithm to use for chart that can connect two points (e.g.
+ * series items_.
```

Comment:
Closing parenthesis misising.
```suggestion
 * series items).
```